maestro-cli/src/main/java/maestro/cli/command/TestCommand.kt

```diff
@@ -642,4 +657,44 @@ class TestCommand : Callable<Int> {
             totalTests = sumOf { it.totalTests ?: 0 }
         )
     }
+
+    private fun showCloudFasterResultsPromotionMessageIfNeeded() {
+        val promotionStateManager = PromotionStateManager()
+        val today = LocalDate.now().toString()
+        
+        // Don't show if already shown today
+        if (promotionStateManager.getLastShownDate("fasterResults") == today) {
+            return
+        }
+        
+        // Don't show if user has used cloud command within last 3 days
+        if (promotionStateManager.wasCloudCommandUsedWithinDays(3)) {
```

Comment:
Why we should even show if user is using maestro cloud command. I think we should not show this if they are cloud user.

Reason: Upselling of cloud should be part of cloud and not CLI for paid users.